templates/publisher/publicise/github_badges.html

```diff
@@ -1,5 +1,37 @@
 {% extends 'publisher/publicise/publicise_layout.html' %}
 
 {% block publicise_content %}
-  <h4>Github badges page</h4>
+  <h4>Promote your charm using an embeddable GitHub badge</h4>
+  <div class="row">
+    <div class="col-2"></div>
```

Comment:
Can you use the [empty columns from Vanilla](https://vanillaframework.io/docs/patterns/grid#empty-columns).

Reply:
yes, makes sense